frontend/src/libs/helpers/init-debounce/init-debounce.ts

```diff
@@ -0,0 +1,21 @@
+import { type FunctionToDebounce } from 'shared/build/index.js';
+
+const initDebounce = <T extends unknown[], R>(
```

Comment:
move debounce to the shared (since we will need on the both, frontend and backend sides)